File: printMegaBytesAndKiloBytes/printMegaBytesAndKiloBytes.java
package com.diwakar;

public class Main {

    public static void main(String[] args) {
        printMegaBytesAndKiloBytes(2050);
    }
    public static void printMegaBytesAndKiloBytes(int kiloBytes) {
        int megaBytes;
        int convertedKB;
        if (kiloBytes<0){
            System.out.println("Invalid Value");
        }else if(kiloBytes<1024){
            System.out.println(kiloBytes+" KB = 0 MB and "+kiloBytes+" KB");
        }else{
            megaBytes = kiloBytes / 1024;
            convertedKB = kiloBytes % 1024;
            System.out.println(kiloBytes+" KB = "+megaBytes+" MB and "+convertedKB+" KB");
        }
    }

}

File: printMegaBytesAndKiloBytes/printMegaBytesAndKiloBytes.js
function printMegaBytesAndKiloBytes(kiloBytes) {
  if (kiloBytes<0){
      console.log("Invalid Value");
  }else if(kiloBytes<1024){
      console.log(kiloBytes+" KB = 0 MB and "+kiloBytes+" KB");
  }else{
      megaBytes =Math.floor(kiloBytes / 1024);
      convertedKB = kiloBytes % 1024;
      console.log(kiloBytes+" KB = "+megaBytes+" MB and "+convertedKB+" KB");
  }
}

printMegaBytesAndKiloBytes(2012);
